PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -217,7 +217,12 @@ def miniAOD_customizeCommon(process):
                   'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_50ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_25ns_nonTrig_V2p1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff']
-    switchOnVIDPhotonIdProducer(process,DataFormat.MiniAOD) 
+    switchOnVIDPhotonIdProducer(process,DataFormat.AOD) 
+    process.egmPhotonIsolation.srcToIsolate = \
+        cms.InputTag("reducedEgamma","reducedGedPhotons")  
+    for iPSet in process.egmPhotonIsolation.isolationConeDefinitions:
+        iPSet.particleBasedIsolation = cms.InputTag("reducedEgamma","reducedPhotonPfCandMap")    
```

Comment:
@slava77 I have checked the dump of miniAOD configuration and it doesn't show that there is a change in the event content.